### DISCUSSION TITLE
BlueMap Support Plugin.

### DISCUSSION THREAD

**Discussion by Lythium4848** [GriefPrevention/GriefPrevention#1269] [Ideas]
Would it be possible to add support for the BlueMap plugin. It would be similar to the Dynmap plugin but would work on BlueMap Instead.

Bluemap: https://www.spigotmc.org/resources/bluemap.83557/
Dynmap Plugin: https://dev.bukkit.org/projects/dynmap-griefprevention

**Comment by samdev-7** [comment]
I would love if this were to be a feature! Here's the api https://github.com/BlueMap-Minecraft/BlueMapAPI

**Reply by RoboMWM** [reply]
Yes, an addon can implement support just like there are dynmap addons that do the same thing.
